src/api/interactivity.js

```diff
@@ -11,29 +71,68 @@ const EVENTS = [
     'featureLeave',
 ];
 export default class Interactivity {
+    /**
+    *
+    * Interactivity purpose is to allow the reception and management of user-generated events, like clicking, over layer features.
+    *
+    * To create a Interactivity object an array of {@link carto.Layer} is required.
+    * Events fired from interactivity objects will refer to the features of these layers and only these layers.
+    *
+    * @param {Array<carto.Layer>} layerList - Array of {@link carto.Layer}, events will be fired based on the features of these layers. The array cannot be empty, and all the layers must be attached to the same map.
+    *
+    * @example
+    * const myLayer = new carto.Layer('layer0', mySource, myStyle);
```

Comment:
this example is wrong, we must wait for the . `layer.loaded` event to create the interactivity.

In the future, I think we should make the addtoMap async

Reply:
Yes

Reply:
Fixed